x-pack/plugins/endpoint/common/types.ts

```diff
@@ -157,6 +157,20 @@ export interface EndpointEvent {
   };
 }
 
+/**
+ * Whitelist Rule
+ * https://github.com/elastic/endpoint-dev/issues/6532
+ */
+export interface WhitelistRule {
+  eventTypes: string[];
+  whitelistRuleType: "simple";
+  whitelistRule: {
+    type: string;
```

Comment:
This could be more specific. `"equality"`.

Reply:
Yeah I think there are other terms it could be as well. Ill straighten it out.